src/components/homepage.js

```diff
@@ -21,25 +22,10 @@ const HomePage = () => {
   const query = new URLSearchParams(location.search);
   const search = query.get('search') || '';
   Africa = Africa.filter((country) => country.country.includes(search.toLowerCase()));
-  const navigate = useNavigate();
-  const [searchValue, setSearchValue] = useState(search);
-
-  const countryFilterOnChange = (event) => {
-    navigate(event.target.value ? `?search=${event.target.value}` : '');
-    setSearchValue(event.target.value);
-  };
 
   return (
     <div className="homePage">
       <h1 className="continent">Africa</h1>
-      <form className="formSearch">
-        <div>
-          <FaSearchLocation />
-        </div>
-        <div>
-          <input className="searchInput" type="text" value={searchValue} placeholder="Search for country..." onChange={countryFilterOnChange} />
```

Comment:
- [ ] (OPTIONAL) Your app works okay now but it would be better if you could find a way to add the search bar and filter your countries by name. This will improve the UI/UX of your site 💯